### PR TITLE
Issues #129 #132: Added redirect to docs for latest launchpad template

### PR DESCRIPTION
**Pull request by rhoads-zach**

### CODE REVIEW

**Comment by rhoads-zach** [comment]
@ALRubinger take a look and let me know what you think

File: ci/openshiftio-appdev-docs/src/main/java/io/openshift/appdev/documentation/builder/HttpApplication.java [hunk 1]
```diff
@@ -24,13 +24,15 @@
 public class HttpApplication extends AbstractVerticle {
 
    private static final String INDEX_PAGE = "index.html";
-
+   private static final String LAUNCHPAD_TEMPLATE_LATEST_URL = "https://raw.githubusercontent.com/openshiftio/launchpad-templates/v3/openshift/launchpad-template.yaml";
```
Comment (cescoffier) [inline ci/openshiftio-appdev-docs/src/main/java/io/openshift/appdev/documentation/builder/HttpApplication.java]: Do we want to get this value from an external configuration (file, or whatever) ?

Comment (ALRubinger) [inline ci/openshiftio-appdev-docs/src/main/java/io/openshift/appdev/documentation/builder/HttpApplication.java]: Yes, but let's deal with that in #142

**Comment by ALRubinger** [comment]
d8506d013cffafc073861be8864a1b2aafb8f7f9

**Pull request closed without merging**